CHANGELOG.md

```diff
@@ -80,6 +80,8 @@
 ### WebView
 Fixed issue where custom URI schemes were matched too greedily in URLs, making for erroneously intercepted URL requests.
 
+- Exported the methods goBack, goForward, reload and stop for use in FuseJS
+
```

Comment:
This is in the wrong section of the changelog. This feature is not part of 1.3, which has been feature-frozen for a while.

Reply:
Just to be clear, this should be put in the "Unreleased"-section of the changelog. I like that you looked for a suitable heading, but sadly there isn't one that we can reuse for this :/